package.json

```diff
@@ -4,7 +4,8 @@
   "description": "an lms for use by codefellows",
   "main": "server.js",
   "scripts": {
-    "test": "./node_modules/mocha/bin/mocha test"
+    "pretest": "./node_modules/gulp/bin/gulp.js test",
+    "test": "./node_modules/mocha/bin/mocha test && ./node_modules/jest-cli/bin/jest.js"
```

Comment:
Updated scripts so it will run `gulp test` prior to test and runs both `mocha` and `jest` once the `gulp` is done.